apis/database/v1alpha2/rdsinstance_types.go

```diff
@@ -129,21 +768,318 @@ const (
+	// Amazon CloudWatch Logs log stream that receives the Enhanced Monitoring
+	// metrics data for the DB instance.
+	EnhancedMonitoringResourceArn string `json:"enhancedMonitoringResourceArn,omitempty"`
+
+	// LatestRestorableTime specifies the latest time to which a database can be
+	// restored with point-in-time restore.
+	LatestRestorableTime metav1.Time `json:"latestRestorableTime,omitempty"`
+
+	// OptionGroupMemberships provides the list of option group memberships for this DB instance.
+	OptionGroupMemberships []OptionGroupMembership `json:"optionGroupMemberships,omitempty"`
+
+	// PendingModifiedValues specifies that changes to the DB instance are pending. This element is only
+	// included when changes are pending. Specific changes are identified by subelements.
+	PendingModifiedValues PendingModifiedValues `json:"pendingModifiedValues,omitempty"`
+
+	// PerformanceInsightsEnabled is true if Performance Insights is enabled for
+	// the DB instance, and otherwise false.
+	PerformanceInsightsEnabled bool `json:"performanceInsightsEnabled,omitempty"`
+
+	// ReadReplicaDBClusterIdentifiers contains one or more identifiers of Aurora DB clusters to which the RDS DB
+	// instance is replicated as a Read Replica. For example, when you create an
+	// Aurora Read Replica of an RDS MySQL DB instance, the Aurora MySQL DB cluster
+	// for the Aurora Read Replica is shown. This output does not contain information
+	// about cross region Aurora Read Replicas.
+	ReadReplicaDBClusterIdentifiers []string `json:"readReplicaDBClusterIdentifiers,omitempty"`
+
+	// ReadReplicaDBInstanceIdentifiers contains one or more identifiers of the Read Replicas associated with this
+	// DB instance.
+	ReadReplicaDBInstanceIdentifiers []string `json:"readReplicaDBInstanceIdentifiers,omitempty"`
+
+	// ReadReplicaSourceDBInstanceIdentifier contains the identifier of the source DB instance if this DB instance is
+	// a Read Replica.
+	ReadReplicaSourceDBInstanceIdentifier string `json:"readReplicaSourceDBInstanceIdentifier,omitempty"`
+
+	// SecondaryAvailabilityZone specifies the name of the secondary Availability Zone for a DB
+	// instance with multi-AZ support when it is present.
+	SecondaryAvailabilityZone string `json:"secondaryAvailabilityZone,omitempty"`
+
+	// StatusInfos is the status of a Read Replica. If the instance is not a Read Replica, this
+	// is blank.
+	StatusInfos []DBInstanceStatusInfo `json:"statusInfos,omitempty"`
+
+	// VPCSecurityGroups provides a list of VPC security group elements that the DB instance belongs
+	// to.
+	VPCSecurityGroups []VPCSecurityGroupMembership `json:"vpcSecurityGroups,omitempty"`
+}
+
+// An RDSInstanceStatus represents the observed state of an RDSInstance.
+type RDSInstanceStatus struct {
+	runtimev1alpha1.ResourceStatus `json:",inline"`
+	AtProvider                     RDSInstanceObservation `json:"atProvider,omitempty"`
 }
 
 // +kubebuilder:object:root=true
```

Comment:
I believe you'll need to enable the status subresource comment marker to `RDSInstance` below in order for the managed resource reconciler to work.